eternalcore-core/src/main/java/com/eternalcode/core/feature/vanish/VanishPermissionConstant.java

```diff
@@ -0,0 +1,6 @@
+package com.eternalcode.core.feature.vanish;
+
+class VanishPermissionConstant {
+
+    static final String VANISH_SEE_PERMISSION = "eternalcore.vanish.see";
```

Comment:
W sumie po co ta klaska jak i tak używasz tego w 1 pliku

Reply:
Planuje w przyszłości rozbudować system vanisha, a czemu teraz to dodałem? Idk niech se będzie, lubię jak permisję są fajnie wydzielone

Reply:
Oki